hedyorg/hedy
# [UI] Fix highlighting of return command and list accesses

**Description**

* Fixes the incorrect name that was used to get the `return` value from the highlighting json
* Adds the `Int` and `Number` rule to ListAccess when appropriate. Functionally this does nothing since they were still being highlighted, but now the trees we generate don't have any error nodes.
* Allows use of list accesses on left hand side of assignments

**Fixes #4857 and fixes #4856 **

**How to test**

Go to level 12 and write this program:

```
define sum with n1, n2
    return n1 + n2
```
Check that return is being highlighted as a keyword

Now go to level 16 and type:
```
print l at [1]
```
The closing bracket should be highlighted

Go to level 12 and type:

```
list at number = 1
```
The variables and the at should be highlighted
